cibyl/sources/zuuld/backends/aggr.py

```diff
@@ -0,0 +1,66 @@
+"""
+#    Copyright 2022 Red Hat
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+"""
+import logging
+from typing import Generic, Iterable
+
+from overrides import overrides
+
+from cibyl.sources.zuuld.backends.abc import T, ZuulDBackend
+from cibyl.sources.zuuld.models.job import Job
+
+LOG = logging.getLogger(__name__)
+
+
+class AggregatedBackend(Generic[T], ZuulDBackend[T]):
```

Comment:
This one is unused right now, but once the GitHub implementation is up, this will allow both implementations to cooperate and provide maximum support.